editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -98,6 +98,28 @@ const Feedback: React.FC = () => {
     }
   }, [breadcrumbs]);
 
+  const feedbackComponentRef = useRef<HTMLDivElement | null>(null);
+
+  const shouldScrollToView = () => {
+    switch (currentFeedbackView) {
+      case "banner":
+        return false;
+      case "thanks":
+        return false;
```

Comment:
```suggestion
      case "banner":
      case "thanks":
        return false;
```

Reply:
nit: Sorry - small nitpick but a bit more readable / consistent with rest of codebase.